app/main.py

```diff
@@ -1,6 +1,26 @@
+import os
+
+import requests
+
+API_KEY = os.getenv("API_KEY")
+URL = "https://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    print("Performing request to Weather API for city Paris...")
+    response = requests.get(URL,
+                            params={"key": API_KEY, "q": FILTERING})
+
+    if response.status_code != requests.codes.ok:
+        print(f"Request failed. Status code: {response.status_code}. "
+              f"Message: {response.text}")
+        return
```

Comment:
in general we use if status_code != 200, because it's GET endpoint that returns either 200 code(success) or 4xx 5xx codes(fail), but this approach is ok as well